to-do-list/src/components/Navi.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+
+interface Props {
+
+}
+
+interface State {
+    naviList: Array<{ id: number, title: string }>
+}
+
+class Navi extends Component<{}, State>{
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            naviList: [
+                { id: 1, title: '전체' },
+                { id: 2, title: '할 일' },
+                { id: 3, title: '완료한 일' },
+                { id: 4, title: '즐겨찾기' }
+            ]
+        }
+    }
+
+    render(): JSX.Element {
+
+        const data = this.state.naviList;
+        const navElementlist = data.map(item => {
+            return (
+                <li key={item.id}><a href={'/list/' + item.id}>{item.title}</a></li>
```

Comment:
```suggestion
                <li key={item.id}><a href={`/list/${item.id}`}>{item.title}</a></li>
```

es6부터 template literal 이라는 문법을 사용해서 다이나믹한 문자열 조합을 간결하게 만들 수 있습니다.

e.g. https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Template_literals